hwdb.d/60-evdev.hwdb

```diff
@@ -434,6 +434,13 @@ evdev:name:Synaptics TM3289-002:dmi:*svnLENOVO*:pvrThinkPadX1Carbon5th*
  EVDEV_ABS_35=::19
  EVDEV_ABS_36=::19
 
+# Lenove Thinkpad X1 Tablet Gen3.
+evdev:input:b0003v17EFp60B5*
+ EVDEV_ABS_00=0:1163:12
```

Comment:
judging by the pastebin output I saw earlier for this device, the axis ranges are correct, it's just the resolution that is off. Let's use the shortened format like in the entry above where `0:1163:12` becomes just `::12`. Same with the other axes please.

Reply:
Done